lib/cluster/s3_distribution_bucket.rb

```diff
@@ -74,7 +73,7 @@ def self.object_keys_to_delete(object)
       end
     end
 
-    def self.default_bucket_policy
+    def self.default_bucket_policy_for(bucket_name)
```

Comment:
@rute-santos this is probably where we'd want to change the bucket policy depending on whether or not it's for distribution or to serve as a file archive.